### PR TITLE
feat [8693u2znn]: docker compose file

### PR DESCRIPTION
**Pull request by LyubovSugak**
### Local development

For convinient development locally was created and set up docker compose file. It's a solution also to avoid CORS error.

#### Important! 
Repos Web FE and Auth should be located in the same directory, otherwise you need to adapt values `volumes` in docker-compose file accordingly.

`docker-compose up` is command to run the project using docker-compose file.

### Connection to cloud DB

Connection Mongo DB string to use for production purposes
To connect to the cloud database change value of env variable STORAGE_CONNECTION_STRING. 
In `docker-compose.yml`:
```
    - STORAGE_CONNECTION_STRING=mongodb+srv://wefriiends-backup:wefriiends2023@cluster0.wir50id.mongodb.net/authorization?retryWrites=true&w=majority
```

Readme file was updated

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+version: '3.8'
+
+services:
+  auth:
+    image: node:latest
```
Comment (Svitlana-G) [inline docker-compose.yaml:5]: May be better to use a specific version?

File: docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+  storage:
+    image: mongo:latest
+    ports:
+      - "27017:27017" # Default MongoDB port
+    volumes:
+      - mongodb_data:/data/db
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=wefriiends
+      - MONGO_INITDB_ROOT_PASSWORD=women2022
```
Comment (Svitlana-G) [inline docker-compose.yaml:42]: I don't think it's a good idea to write them openly

File: README.md [hunk 3]
```diff
@@ -79,4 +79,22 @@ customization MUI styles - default font is Inter, primary color
...
\ No newline at end of file
+file with generic/common app styles (Titles, buttons etc.)
+
+### `Local development`
+
+For convinient development locally was created and set up docker compose file. It's a solution also to avoid CORS error.
+
+#### Important! 
+Repos Web FE and Auth should be located in the same directory, otherwise you need to adapt values `volumes` in docker-compose file accordingly.
+
```
Comment (NataliaDoudkina) [inline README.md:90]: We need Auth in the mobile app too, so we will keep it together with the FE and adapt values for Mobile App? Is it possible? OR create values for both and leave it outside of both apps?

File: docker-compose.yaml (hunk 2 above)
Comment (LyubovSugak) [reply docker-compose.yaml:42]: @Svitlana-G credentials are not from cloud DB, it's locally created

File: docker-compose.yaml (hunk 1 above)
Comment (LyubovSugak) [reply docker-compose.yaml:5]: i don't have strong opinion, GPT helped to create this file

File: README.md (hunk 3 above)
Comment (LyubovSugak) [reply README.md:90]: if you check paths in `values` you can see two repos should be located in the same directory. That's all. For the mobile - we need the same file there

Comment (NataliaDoudkina) [reply README.md:90]: Ok, I don't really understand what is going on here, so I'll trust you lol

**Review by NataliaDoudkina — APPROVED** [review]
Don't quite understand the process, will need some time to learn lol
Can you add this to mobile backup repo please?

**Pull request merged**